### PR TITLE
fix(react): Make TwoColumnPanel wrap to prevent content from getting cut out

### PR DESCRIPTION
**Pull request by dequejosie**
closes https://github.com/dequelabs/cauldron/issues/786

### CODE REVIEW

File: packages/styles/two-column-panel.css [hunk 1]
```diff
@@ -176,12 +176,16 @@
   margin-bottom: 0;
 }
 
+.TwoColumnPanel__ButtonToggle {
+  display: flex;
+}
```
Comment (dequejosie) [inline packages/styles/two-column-panel.css:181]: make ButtonToggle the same height as header

**Review by scurker — DISMISSED** [review]
Great work @dequejosie!

Just one small tweak is needed to make the padding consistent when it wraps. Since the line-height was removed, we need to set the padding for `.TwoColumnPanel__Header` so it doesn't look squished when content wraps:

```css
padding: calc(var(--space-small) - 2px);
```

The `-2px` is to account for the borders that are outside of the element and will make it line up with the left panel when there's only one line of text.

File: packages/styles/two-column-panel.css [hunk 2]
```diff
@@ -176,12 +177,16 @@
...
+
 .TwoColumnPanel__ButtonToggle button,
 .TwoColumnPanel__Close button {
   background-color: transparent;
   border: none;
   border-bottom: 1px solid var(--two-column-panel-border-color);
-  height: var(--two-column-panel-header-height);
+  min-height: var(--two-column-panel-header-height);
```
Comment (dequejosie) [inline packages/styles/two-column-panel.css:189]: This does not change Close button, so I just added here to make Toggle button to expand.

**Comment by dequejosie** [comment]
Screen Shot 2023-03-15 at 11 31 59 AM

**Review by scurker — APPROVED** [review]

**Pull request merged**